sdcm/docker.py

```diff
@@ -141,8 +141,13 @@ def __init__(self, **kwargs):
 
     def _create_node_image(self):
         self._update_image()
+        private_key = self.credentials[0].key_file
```

Comment:
Why this is not done as part of the image Dockerfile, why do we need to create this here ?